hackathon_site/dashboard/frontend/src/components/orders/OrdersTable/OrdersTable.tsx

```diff
@@ -0,0 +1,164 @@
+                    <img
+                        src={ReturnedIcon}
+                        className={styles.statusIcon}
+                        alt={`${status} icon`}
+                    />
+                ) : status === "Pending" ? (
+                    <img
+                        src={PendingIcon}
+                        className={styles.statusIcon}
+                        alt={`${status} icon`}
+                    />
+                ) : (
+                    <img
+                        src={InProgressIcon}
+                        className={styles.statusIcon}
+                        alt={`${status} icon`}
+                    />
+                )}
+                {status}
+            </div>
+        </div>
+    );
+};
+
+const handleEvent: GridEventListener<"rowClick"> = (
+    params, // GridRowParams
+    event, // MuiEvent<React.MouseEvent<HTMLElement>>
+    details // GridCallbackDetails
+) => {
+    console.log("clicked", params.row);
+};
+
+const OrdersTable = ({ ordersData }: OrdersTableProps) => {
+    const columns: GridColDef[] = [
+        { field: "id", headerName: "ID", width: 25, flex: 1 },
+        {
+            field: "created_at",
+            headerName: "Time Ordered",
+            flex: 1,
+            valueFormatter: (params) => {
+                const time = params.value as string;
+                const date = parseISO(time);
+                const formattedTime = format(date, "MMM d, HH:mm");
+                return formattedTime;
+            },
+        },
+        { field: "team_code", headerName: "Team", flex: 1 },
+        {
+            field: "items",
+            headerName: "Order Qty",
+            flex: 1,
+            valueGetter: (params) => {
+                let orderQty: number = 0;
+                const items = params?.value as ItemsInOrder[];
+                try {
+                    orderQty = items.length;
+                } catch (err) {
+                    return 0;
+                }
+                return orderQty;
+            },
+        },
+        {
+            field: "status",
+            headerName: "Status",
+            minWidth: 250,
+            renderCell: (params) => <OrderStateIcon status={params.value} />,
+        },
+        { field: "team_id", headerName: "Team ID" },
+        { field: "updated_at", headerName: "Updated At" },
+        { field: "request", headerName: "Request" },
+    ];
+
+    return (
+        <>
+            <div style={{ width: "100%", height: "700px" }}>
+                <DataGrid
+                    rows={ordersData}
+                    columns={columns}
+                    // pageSize={10} // not available in free version
+                    autoPageSize={true} // adjusts page size to fit available area
+                    pageSizeOptions={[5, 10, 15]}
+                    columnVisibilityModel={{
+                        // hide specific columns
+                        team_id: false,
```

Comment:
currently hides these fields of the Order type, we can display more as well if we need!

Reply:
We probably want to keep team id visible